configure.ac

```diff
@@ -47,6 +47,13 @@ AC_PROG_INSTALL
 AM_GNU_GETTEXT([external])
 AM_GNU_GETTEXT_VERSION([0.19.8])
 
+dnl Define python version
```

Comment:
Since the python scripts are only used for testing, I would rather not adding this kind of flexibility. The users could supply the `PYTHON` envvar to `configure`.